Makefile

```diff
@@ -140,3 +140,7 @@ dist-clean:
 
 calibnet: GOFLAGS+=-tags=calibnet
 calibnet: build
+
+.PHONY: test
+test:
+	go test $(GOFLAGS) -v ./...
```

Comment:
Can you clarify what this does?

Reply:
this is a convenience make target, so you can type less characters :) it doesn't bear much importance to this PR, but will be helpful when we add more tests soon.

make targets by default are files on the filesystem -- the `.PHONY` pseudo-target tells make that `test` doesn't actually refer to a file on the filesystem, i.e. it is a virtual target

i.e
`$ make test`
instead of
`$ go test -v ./...`